### PR TITLE
Goerli Deployment Adjustments

### PR DESCRIPTION
**Pull request by 0xrin1**
- resolve goerli deployment simulation bug
- add in constants for mock WETH and USDC

### CODE REVIEW

File: script/DeployLeveragedEthTestnet.s.sol [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
     function deployMockTokens() internal override {
-        weth = new MockWETH();
-        usdc = MockUSDC(0x30e2B7a907997fDC5a71E377Ece54FAae9F5392D);
+        weth = MockWETH(payable(C.WETH));
+        usdc = MockUSDC(C.USDC);
     }
 
     function run() external {
-        deployMockTokens();
-        deployMocks();
```
Comment (joaop21) [inline script/DeployLeveragedEthTestnet.s.sol:20]: I get why you are removing `deployMockTokens()`, but it's also needed to remove `deployMocks()`?

File: script/DeployLeveragedEthLocal.s.sol [hunk 2]
```diff
@@ -27,8 +27,6 @@ contract DeployScript is DeployLeveragedEth, Test {
             return;
         }
 
-        deployMockTokens();
-        deployMocks();
```
Comment (joaop21) [inline script/DeployLeveragedEthLocal.s.sol:31]: same here

File: script/DeployLeveragedEthTestnet.s.sol (hunk 1 above)
Comment (0xrin1) [reply script/DeployLeveragedEthTestnet.s.sol:20]: because i noticed after the fact that deploy executes deployMocks also

Comment (0xrin1) [reply script/DeployLeveragedEthTestnet.s.sol:20]: like after the first time i introduced this structure for local and test

Comment (joaop21) [reply script/DeployLeveragedEthTestnet.s.sol:20]: I get it. I just went to see the code. Makes sense.

**Review by joaop21 — APPROVED** [review]

**Pull request merged**